PluginExampleProject/src/main/scala/com/alpine/plugin/samples/ver1_0/DBConfusionMatrix.scala

```diff
@@ -0,0 +1,226 @@
+    new OperatorMetadata(
+      name = "Sample - DB Confusion Matrix",
+      category = "Plugin Sample - DB",
+      author = "Jenny Thompson",
+      version = 1,
+      helpURL = "",
+      iconNamePrefix = ""
+    )
+  }
+}
+
+class DBConfusionMatrixGUINode extends OperatorGUINode[
+  Tuple2[ClassificationModelWrapper, DBTable],
+  DBTable] {
+  override def onPlacement(operatorDialog: OperatorDialog,
+                           operatorDataSourceManager: OperatorDataSourceManager,
+                           operatorSchemaManager: OperatorSchemaManager): Unit = {
+    //add parameters to let the user determine how the output table will be written
+    DBParameterUtils.addStandardDatabaseOutputParameters(operatorDialog, operatorDataSourceManager)
+    operatorSchemaManager.setOutputSchema(ConfusionMatrixUtils.outputTabularSchema)
+  }
+
+}
+
+object ConfusionMatrixUtils {
+
+  val observedColumnName: String = "Observed"
+  val predictedColumnName: String = "Predicted"
+  val countColumnName: String = "N" // Don't use "count" as it is a SQL keyword.
+
+  val outputTabularSchema = TabularSchema(
+    Seq(
+      ColumnDef(observedColumnName, ColumnType.String),
+      ColumnDef(predictedColumnName, ColumnType.String),
+      ColumnDef(countColumnName, ColumnType.String)
+    )
+  )
+
+}
+
+class DBConfusionMatrixRuntime extends DBRuntime[Tuple2[ClassificationModelWrapper, DBTable], DBTable] {
+
+  override def onExecution(context: DBExecutionContext,
+                           input: Tuple2[ClassificationModelWrapper, DBTable],
+                           params: OperatorParameters,
+                           listener: OperatorListener): DBTable = {
+    //get the output parameters
+    val outputSchema = DBParameterUtils.getDBOutputSchemaParam(params)
+    val isView = DBParameterUtils.getIsViewParam(params)
+    val outputName = DBParameterUtils.getResultTableName(params)
+    val connectionInfo = context.getDBConnectionInfo
+
+    //check if there is a table  or with the same name as the output table and drop according to the
+    // "overwrite"
+    val overwrite = DBParameterUtils.getOverwriteParameterValue(params)
+    val fullOutputName = getQuotedSchemaTableName(outputSchema, outputName)
+    if (overwrite) {
+      val stmtTable = connectionInfo.connection.createStatement()
+      //First see if a table of that name exists.
+      // This will throw an exception if there is a view with the output name,
+      // we will catch the exception and delete the view in the next block of code.
+      try {
+        listener.notifyMessage("Dropping table if it exists")
+        val dropTableStatementBuilder = new StringBuilder()
+        dropTableStatementBuilder ++= "DROP TABLE IF EXISTS " + fullOutputName + " CASCADE;"
+        stmtTable.execute(dropTableStatementBuilder.toString())
+      }
+      catch {
+        case (e: Exception) => listener.notifyMessage("A view of the name " + fullOutputName + "exists");
```

Comment:
you are going to want a space before "exists" otherwise that error will look back